ssl/ssl_sess.c

```diff
@@ -246,6 +246,12 @@ SSL_SESSION *ssl_session_dup(SSL_SESSION *src, int ticket)
     return NULL;
 }
 
+SSL_SESSION *SSL_SESSION_dup(SSL_SESSION *src) {
+    if (src == NULL)
```

Comment:
If I remove the NULL check then the ssl_session_dup will potentially segfault since it dereferences it without checking validity.  Can I add a check in ssl_session_dup that does a goto err;?